Ginger/GingerCore/Actions/Common/ActUIElement.cs

```diff
@@ -136,7 +136,7 @@ public new static partial class Fields
             //used for MouseClickAndValidate
             public static string ClickType = "ClickType";
             public static string ValidationType = "ValidationType";
-            public static string ValidationElementType = "ValidationElement";
+            public static string ValidationElementType = "ValidationElementType";
```

Comment:
you need to change it back to be "ValidationElement", you can't change this field name- it will break existing users flows